src/services/doppler-api-client.ts

```diff
@@ -143,6 +143,54 @@ export class HttpDopplerApiClient implements DopplerApiClient {
     }));
   }
 
+  private getUnsubscribeStatus(unsubscriptionType: string): string {
+    let unsubscribeStatus: string = '';
+    switch (unsubscriptionType) {
+      case 'hardBounce':
+        unsubscribeStatus = 'unsubscribe_by_hard';
+        break;
+      case 'softBounce':
+        unsubscribeStatus = 'unsubscribe_by_soft';
+        break;
+      case 'neverOpen':
+        unsubscribeStatus = 'unsubscribe_by_never_open';
+        break;
+      case 'manual':
+        unsubscribeStatus = 'unsubscribe_by_client';
```

Comment:
Here in `manual` you should check `manualUnsubscriptionReason` property also if ti is `administrative` is `unsuscribed_by_client` else is `unsuscribed_by_subscriber` you can check the code here: https://github.com/MakingSense/Doppler/blob/develop/Doppler.HypermediaAPI/ApiMappers/FromDoppler/Subscriber_To_Subscriber.cs#L241

Reply:
changed